app/models/druid_version_zip.rb

```diff
@@ -137,8 +140,18 @@ def zip_storage
 
   private
 
+  # should throw an error if any of the files in the moab are not yet readable, for example due to
+  # Ceph MDS instance for a pres cat worker VM thinking that a file is a stray as a result of our
+  # particular use of hardlinking in preservation ingest.  allows for a quick directory walk before
+  # attempting to create the zip file(s).  see https://github.com/sul-dlss/preservation_catalog/issues/1633
+  # @raise [StandardError] if storage_location is not available (should have been provided in constructor)
+  # @raise [Errno::EACCES, ?] if it is not possible to stat one or more files in the Moab
+  def check_moab_version_readability!
+    moab_version_files.map { |f| File.stat(f) }
+  end
+
   def zip_size_ok?
-    total_part_size >= moab_size
+    total_part_size >= moab_version_size
```

Comment:
Really should be `>` here; the zip overhead dictates that the sum of all parts *must* be larger than moab_version_size on disk.

Reply:
good call, done